cypress/e2e/URLShortenerAPI/pricing.cy.js

```diff
@@ -0,0 +1,66 @@
+import { SHORTURLAPI_BASE_URL } from '../../resources/data';
+
+describe('URL Shortener API Pricing page', () => {
+    before(() => {
+        cy.visit(SHORTURLAPI_BASE_URL);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        // Check if the "Accept Cookies" button is visible or exists
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to check "URL Shortener API" title is present', () => {
+        cy.componentVisiblityCheck('.h2', 'URL Shortener API');
+    });
+
+    it('3. Test to check "Starter Plan" is present', () => {
+        cy.componentVisiblityCheck(':nth-child(1) > .card > .card-header > .h3', 'Starter Plan');
+    });
+
+    it('4. Test to check price of "Starter Plan" is present', () => {
+        cy.componentVisiblityCheck(':nth-child(1) > .card > .card-header > .mb-3 > .plan-price', '$1.99');
+    });
+
+    it('5. Test to click on "Subscribe" button', () => {
+        cy.get(':nth-child(1) > .card > .card-header > .btn').click();
+        cy.componentVisiblityCheck('.sidebar-content');
+        cy.get('.lead').should('contains.text', 'Sign in to APILayer');
+        cy.get('body').click(0, 0);
```

Comment:
what are we trying to do here?

Reply:
we are clicking on subscribe button and on click of that button 
![image](https://github.com/user-attachments/assets/a5ffc4e8-839d-4b81-962e-c59489ba832f)
this popup will come so i am checking if popup related content is present or not then i am clicking outside of popup so that popup will go and we can come back to main page